vumi/middleware/manhole.py

```diff
@@ -11,6 +14,14 @@
 from twisted.internet.endpoints import serverFromString
 
 
+class ManholeMiddlewareConfig(Config):
+    twisted_enpoint = ConfigText(
```

Comment:
Would it be the URL field type? https://github.com/praekelt/confmodel/blob/develop/confmodel/fields.py#L71